apollo-router/src/plugins/coprocessor.rs

```diff
@@ -47,6 +47,9 @@ use crate::tracer::TraceId;
 
 pub(crate) const EXTERNAL_SPAN_NAME: &str = "external_plugin";
 const POOL_IDLE_TIMEOUT_DURATION: Option<Duration> = Some(Duration::from_secs(5));
+// TODO: are there better extension codes we could use here?
```

Comment:
I think it's important to clearly identify errors. I agree that it might not be ideal to leak this to a client, but I don't think it's a big deal. Knowing that a problem occurred in an EXTERNAL component quickly narrows down problem resolution.

Reply:
works for me! thanks!